content/manager/update-deployment.md

```diff
@@ -219,21 +220,19 @@ workflows:
 
 plugins:
     plugin1:
-        [...]
         install: true
     plugin2:
-        [...]
         install: true
 ```
-#### Groups, Policy Types and Policy Triggers
-Any Change in the top level fields `groups`, `policy_types` and `policy_triggers` is not currently supported as a part of a deployment update blueprint
+#### Groups, policy types and policy triggers
+Any Change in the top level fields `groups`, `policy_types` and `policy_triggers` is not currently supported as a part of a deployment update blueprint.
 
 ### What Can be Updated as a Part of a Deployment Update
 The following can be updated as part of a deployment update, subject to the limitations that were [mentioned above](http://docs.getcloudify.org/3.4.0/manager/update-deployment/#unsupported-changes-in-a-deployment-update).
```

Comment:
Hard coded 3.4.0? I believe there's a better way to add a link to a headline in the current page